src/fft.rg

```diff
@@ -131,7 +200,7 @@ function fft.generate_fft_interface(itype, dtype)
     var f = c.legion_runtime_select_tunable_value(__runtime(), __context(), tunable_id, 0, 0)
     var n = __future(int64, f)
 
-    -- FIXME (Elliott): I thought Regent was supposed to copy on
+    -- (Elliott): I thought Regent was supposed to copy on 
```

Comment:
Instead of removing the FIXME let's check if this is actually still true or not. If it's still true, then this would still be a FIXME. 😃

Reply:
Yes, this still segfaults if I uncomment that line, so added back the 'fixme'